index.html

```diff
@@ -77,8 +77,14 @@ <h1 tabindex="-1">Find a school</h1>
         </div><!-- /.row -->
 
         <p class="disclaimer lesser-text">Information subject to change; by using this, you agree to our <a href="https://education.nsw.gov.au/going-to-a-public-school/finding-a-public-school/terms-of-use" target="_blank">terms of use</a>.</p>
-      </div>
 
+      <p class="disclaimer lesser-text"><a id="specialist-support-click" href="javascript:return false">Information for specialist support</a></p>
+      <div id="specialist-support-info" style="display:none">
+        <p class="disclaimer lesser-text least-text">The department provides a range of specialist support classes in regular and special schools. Access to these classes is through a separate placement panel process. Parents and carers wanting to enrol their child in a specialist support class should use the School Finder to locate their local public school and contact the principal directly. The principal will provide advice about the process to seek enrolment in a specialist support class.</p>
```

Comment:
What is the 'least-text' class? .lesser-text is defined here: https://github.com/CodeforAustralia/school-finder/blob/2388e6930b8fa43fc4b0f73356a589ed357ca0a9/css/style.css but I don't see least-text